.mailmap

```diff
@@ -269,6 +270,7 @@ Michael Rodriguez <michael@newdream.net>
 Michael Rodriguez <michael@newdream.net> <michael@squid.newdream.net>
 Michal Koutný <mkoutny@suse.com>
 Min Chen <minchen@ubuntukylin.com>
+Mingqiao Wu <wumingqiao@inspur.com>
```

Comment:
As per the current version of .mailmap, this is in the incorrect alphabetical order.

Reply:
@joscollin Why? Can you help me change this.Thank you.
First name is 'Mingqiao'.
Last name is 'Wu'.
Thank you very much.

Reply:
I meant, This is the correct order:
```
Min Chen
MingXin Liu
Mingqiao Wu
Mingyue Zhao 
```

I don't know how a capital letter came in between for `MingXin`. But that entry was already existing anyway.